### PR TITLE
docs: add windows instructions to run mobile wallet

### PR DESCRIPTION
**Pull request by alexruzenhack**
### Acceptance Criteria
- Instructions to run mobile wallet on Windows.

### Security Checklist
- [ ] Make sure you do not include new dependencies in the project unless strictly necessary and do not include dev-dependencies as production ones. More dependencies increase the possibility of one of them being hijacked and affecting us.

### CODE REVIEW

File: WINDOWS.md [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+## Install Android Studio
+
+See the [releases](https://developer.android.com/studio) that most fits for your use:
+
+- Windows(64-bit) [android-studio-2021.3.1.17-windows.exe](https://redirector.gvt1.com/edgedl/android/studio/install/2021.3.1.17/android-studio-2021.3.1.17-windows.exe)
+- Windows(64-bit) [android-studio-2021.3.1.17-windows.zip](https://redirector.gvt1.com/edgedl/android/studio/ide-zips/2021.3.1.17/android-studio-2021.3.1.17-windows.zip)
+
+## Configure Android Studio Environment Variables
+
+<aside>
```
Comment (pedroferreira1) [inline WINDOWS.md:15]: What's this? I don't see any formatting in the markdown.

https://github.com/HathorNetwork/hathor-wallet-mobile/blob/0becacf654599fbf8b88859a28054d912ce50021/WINDOWS.md

File: WINDOWS.md [hunk 2]
```diff
@@ -0,0 +1,289 @@
...
+[System.Environment]::SetEnvironmentVariable('ANDROID_SDK_ROOT',"$env:USERPROFILE\AppData\Local\Android\Sdk",[System.EnvironmentVariableTarget]::User)
+```
+
+Set `ANDROID_HOME`
+
+```powershell
+[System.Environment]::SetEnvironmentVariable('ANDROID_HOME',"$env:USERPROFILE\AppData\Local\Android\Sdk",[System.EnvironmentVariableTarget]::User)
+```
+
+Now add the parth for the Android Virtual Devices (AVD), this folder contains your configured devices.
```
Comment (pedroferreira1) [inline WINDOWS.md:33]: path*

File: WINDOWS.md [hunk 3]
```diff
@@ -0,0 +1,289 @@
...
+[https://github.com/intel/haxm](https://github.com/intel/haxm)
+
+- See [restrictions](https://developer.android.com/studio/run/emulator-acceleration#vm-accel-restrictions)
+- See your [configuration](https://developer.android.com/studio/run/emulator-acceleration#vm-windows) for a different set-up
+
+# Install Dependencies and Utilities
+
+## Install Chocolatey
+
+It is a streight foraward package manager sotware for Windows and it will be used in installation instructions that follows. Read the [installation instructions](https://chocolatey.org/install).
```
Comment (pedroferreira1) [inline WINDOWS.md:103]: straight* 

forward*

software*

File: WINDOWS.md [hunk 4]
```diff
@@ -0,0 +1,289 @@
...
+```powershell
+winget install Microsoft.OpenJDK.11
+```
+</details>
+
+The installed JDK should be found at `C:\Program Files\Microsoft\jdk-11.0.16.101-hotspot`.
+
+## Configure JDK environment varible
+
+Add `JAVA_HOME` to your environment varibales targeting your user:
```
Comment (pedroferreira1) [inline WINDOWS.md:137]: variables*

File: WINDOWS.md [hunk 5]
```diff
@@ -0,0 +1,289 @@
...
+On windows you can install with Chocolatey:
+
+```powershell
+choco install make
+```
+</details>
+
+## Install POEDIT
+
+See more about [POEDIT](https://poedit.net/), an utility to manage and edit translations, see [TRANSLATING.md](./TRANSLATING.md)
```
Comment (pedroferreira1) [inline WINDOWS.md:157]: a* utility

File: WINDOWS.md [hunk 6]
```diff
@@ -0,0 +1,289 @@
...
+On windows you can install with Chocolatey:
+
+```powershell
+choco install poedit
+```
+
+However the current available version is ****`2.4.2`,** while downloading from the website you get the most recent version.
+</details>
+
+- or instal by downaload
```
Comment (pedroferreira1) [inline WINDOWS.md:170]: install*

download*

File: WINDOWS.md [hunk 7]
```diff
@@ -0,0 +1,289 @@
...
+$env:PATH += ";${env:ProgramFiles(x86)}\Poedit\GettextTools\bin\;"
+```
+
+Persist the new `PATH` value in your `user` environment:
+
+```powershell
+[System.Environment]::SetEnvironmentVariable('Path',$env:PATH,[System.EnvironmentVariableTarget]::User)
+```
+
+After run the command above restart you PowerShell session, then verify running the following command:
```
Comment (pedroferreira1) [inline WINDOWS.md:190]: your*

File: WINDOWS.md [hunk 8]
```diff
@@ -0,0 +1,289 @@
...
+make i18n
+```
+
+If you get an error like this:
+```
+make: *** [Makefile:34: src/locale/pt-br/texts.po.json] Error 1
+```
+then comment the line 34 from `Makefile` file and run again.
+
+If you get an erro like this:
```
Comment (pedroferreira1) [inline WINDOWS.md:252]: error*

**Review by pedroferreira1 — APPROVED** [review]

**Review by andreabadesso — APPROVED** [review]

**Pull request merged**